### PR TITLE
Update getting started -- feedback welcome

### PR DESCRIPTION
**Pull request by jpjpjp**
As a new Portman user I struggled a bit getting up to speed.  There is SO much info in the readme.   After the getting started, the initial readme jumps into things like portman CLI configuration and Postman/Newman variables, but as a new user I was more interested in learning about what types of tests the tool could make and how to set them up.   It actually took me several days of playing with it, reading the main readme from top to bottom, and accidentally discovering that I was getting contract tests despite being misconfigured, before I began to realize how to process the documentation.

TL:DR - the "good stuff" is really in the examples!

With that in mind here are some suggested changes that might make it easier for other first time users.   I won't be upset if you don't choose to take this PR, or only want parts of it, but this tool is so cool and helpful, I wanted to give back in some way.

Main README 
- In the initial section that describes all the things you can do with Portman, I added "learn more" links that take you right to the relevant example.   (Note -- the items that don't have this link are still puzzling to me!)
-  In the Getting Started section I added  another "OR" that suggests just trying to run Portman without any configuration to get a sense of what it can do.   The first time you see that Postman collection with contract tests is a game changer!   This approach lets a new user get to it very quickly without needing to learn all the configuration options
  - When I discovered that Portman makes contract tests without any configuration at all I thought it was really cool.  I attempted to create a configuration that matches the default configuration and I reference it in my 3rd alternate "Getting Started" option. 

Contract Test examples readme:
- I did a fairly significant update of the readme for the Contract Tests because I think its a great place to start
  - Give a little more description about what a contract test is
  - Update the try it out section so that a user who installs portman can run the samples directly without any other configuration
  - Use a yaml based example since its easier to read and supports inline comments
  -  Specify the operationId just once for all the contract tests in the config instead of once per test
  
I hope this is accepted in the spirit it is offered!

### CODE REVIEW

**Review by nicklloyd — APPROVED** [review]
Love it - we’ve always known the documentation was there but a _lot_ to take in…

If there were more hours in the day, Portman would have its own site that included well laid out docs and a UI tool to click generate a config…

Anyway, your contribution greatly appreciated 🤘🏻

File: README.md [hunk 1]
```diff
@@ -32,29 +32,43 @@ With Portman, you can:
...
 
-1. Install Portman
+1. [Install Portman](#installation)
 2. Copy `.env.example` to `.env` and add environment variables you need available to your collection
 3. Copy/rename and customize each of the \_\_\_\_.default.json config files in the root directory to suit your needs
 4. Start converting your OpenAPI document to Postman
 
+OR
+
+If you have an existing OpenaAPI specification, try running Portman without any special setup to see how it can generate a Postman collection with contract tests with it's default configuration.
```
Comment (Gdewilde) [inline README.md:60]: Typo: OpenaAPI -> OpenAPI

**Review by Gdewilde — CHANGES_REQUESTED** [review]

**Comment by Gdewilde** [comment]
Thanks for putting together this PR. I noticed one small typo.

**Review by thim81 — APPROVED** [review]

**Comment by thim81** [comment]
@jpjpjp Thanks for your contributions; we will merge them in.

As @nicklloyd mentioned, the documentation is extensive, partly because there is so much you can do with simple configurations. We started small, and over time, it grew into a large README with numerous examples, though the overview got lost along the way.

If you have any more suggestions or improvements, please do not hesitate to share them.

@Gdewilde I'll take care of the typo corrections.

**Pull request merged**